pkgs/development/libraries/kde-frameworks/plasma-framework.nix

```diff
@@ -18,4 +18,10 @@ mkDerivation {
     qtquickcontrols2
   ];
   propagatedBuildInputs = [ kpackage kservice qtbase ];
+  # align QtQuick usage with qt5.12, otherwise it will be unable to find certain components
+  # see https://github.com/NixOS/nixpkgs/issues/98536
+  postPatch = ''
+    substituteInPlace src/declarativeimports/plasmaextracomponents/qml/ExpandableListItem.qml \
+      --replace "import QtQuick 2.14" "import QtQuick 2.12"
```

Comment:
Not really qualified to review this, but I'm pretty interested:
- Where does the 2.14 come from?
- Where does the 2.12 come from?
- Will this break on the next `libsForQt5 = libsForQt512` *and* `libsForQt5 = libsForQt514` bump?
- Should this be parametrized on some `${qt5.base.version}` to be more future proof?

Reply:
> Where does the 2.14 come from?

It's what the upstream `plasma-frameworks` expects

>Where does the 2.12 come from?

I got that from other applications which were working

> Will this break on the next libsForQt5 = libsForQt512 and libsForQt5 = libsForQt514 bump?

This is specifically for the 20.09 release, those are unlikely to be bumped.
But it's just the version of the widget, so I don't think this will be too "harmful"

> Should this be parametrized on some ${qt5.base.version} to be more future proof?

AFAIK, these are looked up at runtime, and us needing to use the 5.12 qt libraries is where this mismatch comes from. However, using a "previous" version of the widget seems to be harmless